test/e2e/storage/csi_mock_volume.go

```diff
@@ -244,6 +251,10 @@ var _ = utils.SIGDescribe("CSI mock volume", func() {
 			cs.StorageV1().StorageClasses().Delete(context.TODO(), sc.Name, metav1.DeleteOptions{})
 		}
 
+		for _, vsc := range m.vsc {
+			ginkgo.By(fmt.Sprintf("Deleting volumesnapshotclass %s", vsc.GetName()))
+			m.config.Framework.DynamicClient.Resource(testsuites.SnapshotClassGVR).Delete(context.TODO(), vsc.GetName(), metav1.DeleteOptions{})
+		}
```

Comment:
This is outside of the snapshot feature tag?

Reply:
yeah i kept it here to be consistent with cleaning up of storage classes. Ideally, `vsc` would only be populated in tests that support the feature

Reply:
ok